static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -134,17 +159,39 @@ const addDesktopInlineAds = (isInline1: boolean): Promise<boolean> => {
 
 	const rules = isInline1 ? defaultRules : relaxedRules;
 
+	const enableDebug =
+		(sfdebug === '1' && isInline1) || (sfdebug === '2' && !isInline1);
+
 	const insertAds: SpacefinderWriter = async (paras) => {
+		// Include containers on the non-inline1 pass
+		// i.e. inline2, inline3, etc...
+		const includeContainer = !isInline1;
+
+		// Make ads sticky in containers if using containers and in sticky test variant
+		// Compute the height of containers in which ads will remain sticky
+		const stickyContainerHeights =
+			includeContainer &&
+			isInVariantSynchronous(multiStickyRightAds, 'variant')
+				? computeStickyHeight(paras, articleBodySelector)
+				: undefined;
+
 		const slots = paras
 			.slice(0, isInline1 ? 1 : paras.length)
 			.map((para, i) => {
 				const inlineId = i + (isInline1 ? 1 : 2);
-				const includeContainer = !isInline1;
 
 				if (sfdebug) {
 					para.style.cssText += 'border: thick solid green;';
 				}
 
+				const containerOptions = stickyContainerHeights
```

Comment:
Should this check be `stickyContainerHeights[i]`?

```suggestion
				const containerOptions = stickyContainerHeights[i]
```

Reply:
I might have to work on the naming here, `stickyContainerHeights` has type `number[] | undefined`, and is undefined when we haven't got sticky heights enabled. So we check `stickyContainerHeights` rather than check `stickyContainerHeights[i]` to check if stickiness is enabled.

Reply:
Actually, the follow-up PR that fixes this across breakpoints will address this, so I'm going to leave it here.